src/sagas.js

```diff
@@ -440,27 +498,22 @@ function* startProgress ({ vmId, poolId, name }) {
   }
 }
 
-function* getSingleInstance ({ vmId, poolId }) {
```

Comment:
Why do you remove this function? It fix bug with pool updating.

Reply:
I don't remember doing that, maybe I missed it in a rebase conflict resolution.  I'll put it back.